runtime/jailer_integ_test.go

```diff
@@ -74,18 +74,44 @@ func testJailer(t *testing.T, jailerConfig *proto.JailerConfig) {
 	image, err := alpineImage(ctx, client, defaultSnapshotterName)
 	require.NoError(err, "failed to get alpine image")
 
-	vmID := testNameToVMID(t.Name())
+	fcClient, err := newFCControlClient(containerdSockPath)
+	require.NoError(err)
 
+	vmID := testNameToVMID(t.Name())
+	// additionalDrive is a regular file;
+	// and the following additionalBlockDevice is a block device, aks block special file;
+	// They are used to test different bind mount cases
 	additionalDrive := internal.CreateFSImg(ctx, t, "ext4", internal.FSImgFile{
 		Subpath:  "dir/hello",
 		Contents: "additional drive\n",
 	})
 
+	// default BlockDeviceFile setup, will change UID & GID if a jailerConfig is passed
+	deviceFile := internal.BlockDeviceFile{
+		Subpath: "example/block1",
+		UID:     0,
+		GID:     0,
+		Dev:     259, // 259 is the major number for blkext which is one type of block devices
+	}
+
+	var additionalBlockDevice string
+	if jailerConfig != nil {
+		// cover "With Jailer" & "With Jailer and bind-mount" test cases
+		deviceFile.UID = int(jailerConfig.UID)
+		deviceFile.GID = int(jailerConfig.GID)
+		additionalBlockDevice = internal.CreateBlockDevice(ctx, t, "ext4", deviceFile)
+	} else {
+		// cover "Without Jailer" test case
+		additionalBlockDevice = internal.CreateBlockDevice(ctx, t, "ext4", deviceFile)
```

Comment:
You can have internal.CreateBlockDevice outside of the if block.

Reply:
Fixed.